bin/gen_PKGBUILD.py

```diff
@@ -0,0 +1,44 @@
+import tomli
+
+with open("pyproject.toml", 'rb') as f:
+    pyproject = tomli.load(f)
+
+description = pyproject["project"]["description"]
+version = open('anesthetic/_version.py','r').readlines()[0].split("=")[1].strip().strip("'")
+url = pyproject["project"]["urls"]["Homepage"]
+pyproject["project"]["dependencies"]
+rel=1
+
+
+PKGBUILD = """# Maintainer: Will Handley <wh260@cam.ac.uk> (aur.archlinux.org/account/wjhandley)
+pkgname=python-anesthetic
+_name=${pkgname#python-}
+pkgver=%s
+pkgrel=%s
+pkgdesc="%s"
+arch=(any)
+url="%s"
+license=(MIT)
+groups=()
+depends=(python-numpy python-matplotlib python-scipy python-pandas)
```

Comment:
What about version dependencies such as `pandas>=2.0.0`?
Can this dependencies field be automatically generated from the `pyproject.toml` dependencies?

Reply:
> What about version dependencies such as pandas>=2.0.0?

Unfortunately no. Arch does not version packages, there's only one and that's the one that's current in the repos. As an AUR packager it's your job to ensure that it works.

>  Can this dependencies field be automatically generated from the pyproject.toml dependencies?

There's not a one-to-one correspondence between `pip` packages and AUR packages. Some get renamed unceremoniously.